src/main/java/com/sap/oss/phosphor/fosstars/model/value/ArtifactVersion.java

```diff
@@ -27,19 +27,29 @@
  */
 public class ArtifactVersion {
 
-  public static final ArtifactVersion EMPTY = new ArtifactVersion("", LocalDate.now());
+  /**
+   * Default immutable EMPTY artifact version.
+   */
+  public static final ArtifactVersion EMPTY =
+      new ArtifactVersion("-", LocalDateTime.parse("1970-01-01T01:00:00"));
```

Comment:
Just curious, why did you replace an empty string with a hyphen?

Reply:
I thought that a specific character would be more explicit than just a empty string.
However, I already decided to revert it 🙂